src/reporter/query_1TNENA.py

```diff
@@ -87,6 +87,22 @@ def query_1TNENA(entity_type=None,  # In Path
         logging.getLogger(__name__).error(msg, exc_info=True)
         return msg, 500
 
+    if err == "AggrMethod cannot be applied":
+        r = {
+            "error": "AggrMethod cannot be applied",
+            "description": "AggrMethod cannot be applied on type TEXT and BOOLEAN."
+        }
+        logging.getLogger(__name__).info("AggrMethod cannot be applied")
+        return r, 404
+
+    else:
```

Comment:
This should be the case only if err is not null, right?

Reply:
Actually you can remove the else

Reply:
done